charts/acend-training-chart/values.yaml

```diff
@@ -36,11 +36,11 @@ acendTraining:
       podAnnotations: {}
       ingress:
         annotations:
-          kubernetes.io/ingress.class: nginx-public
           kubernetes.io/tls-acme: "true"
+        ingressClassName: nginx-public
```

Comment:
Our default Ingress class is `nginx` and not anymore `nginx-public`. Can you change this?

But, as you set this now in the defaults of the HelmChart, it will have an affect on all deployments, as the ingressClassName is, as far as I know, never overwritten in a training values.yaml and therefore now set to `nginx-public`
Maybe its better to not define this in the defaults and only leave it as a comment to optionally set it if needed?

Otherwise, by setting this to the correct `nginx`, it should not be a problem. The cluster anyway sets the ingressClassName to `nginx` as its the default IngressClass on the acend Cluster.

Reply:
Changed it to a comment, thanks.